po/es.po

```diff
@@ -30,128 +29,123 @@ msgstr "sosreport (versión %s)"
 
 #: ../sos/sosreport.py:983
-#, fuzzy, python-format
+#, python-format
 msgid "plugin %s requires root permissionsto execute, skipping"
-msgstr "no se instaló el plugin %s, ignorándolo"
+msgstr "el complemento %s necesita privilegios administrativos para ejecutarse; se ha omitido"
 
 #: ../sos/sosreport.py:985
 msgid "requires root"
-msgstr ""
+msgstr "requiere privilegios administrativos"
 
 #: ../sos/sosreport.py:993
 msgid "excluded"
-msgstr ""
+msgstr "excluidos"
 
 #: ../sos/sosreport.py:997
 msgid "skipped"
-msgstr ""
+msgstr "omitidos"
 
 #: ../sos/sosreport.py:1001
 msgid "inactive"
-msgstr ""
+msgstr "inactivos"
 
 #: ../sos/sosreport.py:1005
 msgid "optional"
-msgstr ""
+msgstr "opcionales"
 
 #: ../sos/sosreport.py:1015
 msgid "not specified"
-msgstr ""
+msgstr "sin especificar"
 
 #: ../sos/sosreport.py:1023
-#, fuzzy, python-format
+#, python-format
 msgid "plugin %s does not install, skipping: %s"
-msgstr "no se instaló el plugin %s, ignorándolo"
+msgstr "el complemento %s no se puede instalar; se ha omitido: %s"
 
 #: ../sos/sosreport.py:1027
 #, python-format
 msgid "Unknown or inactive profile(s) provided: %s"
-msgstr ""
+msgstr "Se proveyeron perfiles desconocidos o inactivos: %s"
 
 #: ../sos/sosreport.py:1120
 msgid "no valid plugins found"
-msgstr "no se encontró un plugin válido"
+msgstr "no se encontró ningún complemento válido"
 
 #: ../sos/sosreport.py:1124
 msgid "The following plugins are currently enabled:"
-msgstr "Los siguientes plugins están activados actualmente:"
+msgstr "Los complementos siguientes están activados actualmente:"
 
 #: ../sos/sosreport.py:1130
 msgid "No plugin enabled."
-msgstr "Ningún plugin está activado."
+msgstr "No se ha activado ningún complemento."
 
 #: ../sos/sosreport.py:1134
 msgid "The following plugins are currently disabled:"
-msgstr "Los siguientes plugins no están activados:"
+msgstr "Los complementos siguientes están desactivados actualmente:"
 
 #: ../sos/sosreport.py:1145
 msgid "The following plugin options are available:"
-msgstr "Las siguientes opciones del plugin están disponibles:"
+msgstr "Las opciones del complemento siguientes están disponibles:"
 
 #: ../sos/sosreport.py:1160
 msgid "No plugin options available."
-msgstr "No hay opciones de plugin disponibles."
+msgstr "No hay opciones de complemento disponibles."
 
 #: ../sos/sosreport.py:1172
-#, fuzzy
 msgid "no valid profiles found"
-msgstr "no se encontró un plugin válido"
+msgstr "no se encontró ningún perfil válido"
 
 #: ../sos/sosreport.py:1174
-#, fuzzy
 msgid "The following profiles are available:"
-msgstr "Las siguientes opciones del plugin están disponibles:"
+msgstr "Los perfiles siguientes están disponibles:"
 
 #: ../sos/sosreport.py:1197
 msgid "Press ENTER to continue, or CTRL-C to quit.\n"
 msgstr "Presione INTRO para continuar o CTRL-C para salir.\n"
 
 #: ../sos/sosreport.py:1216
-#, fuzzy
 msgid " Setting up archive ..."
-msgstr "Encriptando el archivo..."
+msgstr " Configurando el archivador…"
```

Comment:
@iranzo The extra space is also in the source string. That should be dealt with in a separate PR.